src/main/resources/ath-container/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Dockerfile to be used to run ATH itself.
+#
+# see docs/DOCKER.md for usage
+#
+
+FROM debian:sid
+MAINTAINER ogondza@gmail.com
+
+RUN apt-get -y update && \
+    apt-get install -y \
+        curl \
+        docker.io \
+        imagemagick \
+        iptables \
+        firefox-esr \
+        maven \
+        openjdk-8-jdk \
+        vnc4server
+
+# Allow injecting uid and git to match directory ownership
```

Comment:
This is sufficiently weird you had better document clearly why this is necessary.